app/web/src/components/FuncEditor/FuncArguments.vue

```diff
@@ -12,54 +12,60 @@
         placeholder="New argument name..."
         :disabled="disabled"
       />
-      <SelectMenu
+      <!-- <SelectMenu
```

Comment:
Fix these selects. Hopefully, we don't have to use `Option` and can mimic other select boxes.